rosidl_typesupport_introspection_cpp/cmake/rosidl_typesupport_introspection_cpp_generate_interfaces.cmake

```diff
@@ -99,7 +99,8 @@ foreach(_pkg_name ${rosidl_generate_interfaces_DEPENDENCY_PACKAGE_NAMES})
     ${_pkg_name})
 endforeach()
 ament_target_dependencies(${rosidl_generate_interfaces_TARGET}${_target_suffix}
-  "rosidl_typesupport_introspection_cpp")
+  "rosidl_typesupport_introspection_cpp"
+  "rosidl_typesupport_introspection_c")
```

Comment:
This should not be necessary since `rosidl_typesupport_introspection_cpp` will already pull in all its dependencies. No need to specify transitive ones explicitly.